src/autometrics/decorator.py

```diff
@@ -42,6 +41,14 @@ def autometrics(
 ):
     """Decorator for tracking function calls and duration. Supports synchronous and async functions."""
 
+    def register_function_info(
+        function: str,
+        module: str,
+    ):
+        get_tracker().initialize_at_zero(
```

Comment:
nit, but i'm not sure how useful it is to have`_at_zero` in the function name. since we are going with lifecycle events for tracker functions i think we can just go with something like `initialize_counters` or `init` and have the comments in the function explain what is happening

Reply:
good point - will change